### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by dhogan8**

### CODE REVIEW

File: src/components/Layout/Footer.tsx [hunk 1]
```diff
@@ -252,7 +252,8 @@ const Footer: React.FC<React.HTMLProps<HTMLDivElement>> = (props) => {
           className={styles.copyright}
         >
           <p>
-            &copy; 2012-
+            &copy;
+            {' '}
             {new Date().getFullYear()}
             {' '}
             MaxMind, Inc. All Rights Reserved.
```
Comment (oschwald) [inline src/components/Layout/Footer.tsx:259]: I think we can drop the "All Rights Reserved." bit as well.

Comment (kevcenteno) [reply src/components/Layout/Footer.tsx]: It's not a big deal.  I could go either way.

Comment (oschwald) [reply src/components/Layout/Footer.tsx]: It is possible that Diana would want us to add that trademark language. I am not sure if that changes anything, i.e.:

> MaxMind, GeoIP, minFraud, and related trademarks belong to MaxMind, Inc.

Comment (oschwald) [reply src/components/Layout/Footer.tsx]: I asked about that in the Slack thread.

Comment (oschwald) [reply src/components/Layout/Footer.tsx]: Oh, I see it on the dev site already. Sorry, I meant only remove the "All Rights Reserved." bit, which is for copyright. We want the trademark language presumably.

**Review by kevcenteno — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
 ### Minimum Requirements
 
-- Node 14
+- Node 16
 - Yarn 1.17.3
```
Comment (kevcenteno) [inline README.md:38]: Yarn should be replaced by npm 8.5.5

**Review by kevcenteno — APPROVED** [review]

**Review by oschwald — APPROVED** [review]
LGTM.

**Pull request merged**